docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -0,0 +1,277 @@
+
+#### YAML snippet
+
+```yaml
+messageCompression: NONE
+```
+
+### zeebe.gateway.threads
+
+| Field             | Description                                                                                                                                                                                           | Example value |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| managementThreads | Sets the number of threads the gateway will use to communicate with the broker cluster. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_THREADS_MANAGEMENTTHREADS`. | 1             |
+
+#### YAML snippet
+
+```yaml
+threads:
+  managementThreads: 1
+```
+
+### zeebe.gateway.security
+
+| Field                | Description                                                                                                                                                      | Example value |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| enabled              | Enables TLS authentication between clients and the gateway. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_ENABLED`. | False         |
+| certificateChainPath | Sets the path to the certificate chain file. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_CERTIFICATECHAINPATH`.   |               |
+| privateKeyPath       | Sets the path to the private key file location. This setting can also be overridden using the environment variable `ZEEBE_GATEWAY_SECURITY_PRIVATEKEYPATH`.      |               |
+
+#### YAML snippet
+
+```yaml
+security:
+  enabled: false
+  certificateChainPath:
+  privateKeyPath:
+```
+
+### zeebe.gateway.longPolling
+
+| Field   | Description                                                                                                                                      | Example value |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| enabled | Enables long polling for available jobs. This setting can also be overridden using the environment `variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED`. | True          |
+
+#### YAML snippet
+
+```yaml
+longPolling:
+  enabled: true
+```
+
+### zeebe.gateway.interceptors
+
+Consider reading our documentation on interceptors first.
+
+Each interceptor should be configured with the values described below:
+
+<table name="interceptors" id="interceptors">
```

Comment:
Is there a reason why this table is structured differently from the others?

Reply:
Laziness and the list elements did not easily convert in markdown.